src/utils/nameUtils.ts

```diff
@@ -21,6 +22,20 @@ export function guardReservedNames(name: string): string {
   return ReservedModelNames.indexOf(name) > -1 ? `${name}Model` : name;
 }
 
+/**
+ * Returns a normalized Type name, this is, the type name capitalized when needed.
+ * Otherwise, return the original typename, for example primitives "string", etc. don' need capitalization
```

Comment:
Nit to show I am paying attention:
```suggestion
 * Otherwise, return the original typename, for example primitives "string", etc. don't need capitalization.
```